pom.xml

```diff
@@ -15,8 +15,8 @@
 
   <properties>
     <aspectj.version>1.9.7</aspectj.version>
-    <raml-module-builder.version>34.1.0</raml-module-builder.version>
-    <vertx.version>4.3.1</vertx.version>
+    <raml-module-builder.version>35.0.0</raml-module-builder.version>
+    <vertx.version>4.3.3</vertx.version>
     <junit.jupiter.version>5.8.2</junit.jupiter.version>
     <rest-assured.version>5.1.1</rest-assured.version>
     <log4j-slf4j-impl.version>2.17.2</log4j-slf4j-impl.version>
```

Comment:
please upgrade to the latest 2.19.0 version

Reply:
fixed